site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -135,34 +135,32 @@ default, Polaris uses an in-memory metastore.
 > The default in-memory metastore is not suitable for production use, as it will lose all data
 > when the server is restarted; it is also unusable when multiple Polaris replicas are used.
 
-To use a durable metastore, you need to switch to the EclipseLink metastore, and provide your own
-`persistence.xml` file. This file contains details of the database used for metastore management and
-the connection settings. For more information, refer to the [metastore documentation]({{% ref
-"metastores" %}}).
+To enable a durable metastore, configure your system to use the Relational JDBC-backed metastore.
+This implementation leverages Quarkus for datasource management and supports configuration through
+environment variables or JVM -D flags at startup. For more information, refer to the [Quarkus configuration reference](https://quarkus.io/guides/config-reference#env-file).
 
-Then, configure Polaris to use your metastore by setting the following properties:
+Configure the metastore by setting the following ENV variables:
 
-```properties
-polaris.persistence.type=eclipse-link
-polaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
-polaris.persistence.eclipselink.persistence-unit=polaris
 ```
+POLARIS_PERSISTENCE_TYPE=relational-jdbc
 
-Where:
+QUARKUS_DATASOURCE_DB_KIND=postgresql
+QUARKUS_DATASOURCE_USERNAME=<your-username>
+QUARKUS_DATASOURCE_PASSWORD=<your-password>
+QUARKUS_DATASOURCE_JDBC_URL=<jdbc-url-of-postgres>
+```
```

Comment:
Both env variable and Quarkus `application.properties` work. Can we be consistent about the configuration in the doc as the metastores.md use the `application.properties` way? I think we can mention the `application.properties` way here while refer to page `https://polaris.apache.org/in-dev/unreleased/configuration/` if users prefer env variable. Thoughts?

Reply:
I support consistency. However, it might be best to do that in another PR.

As to using `application.properties`, as I commented in another thread, they are relative to _current_ dir and as such prone to errors, IMHO.

If we want to go with config files, I propose to revive @jbonofre PR on that.

Reply:
I'm OK with either way. Hi @dimas-b , can you share JB's PR?

Reply:
+1 to revive JB's PR : https://github.com/apache/polaris/pull/1032/files#diff-7f7a4d7e028cff5774c249e58114c43abf6c983ad8be3fc955abec59865272ba

Reply:
I'm also supportive on JB's PR. It provides much better UX in terms of how users config Polaris server and admin tool. User can make changes to `config/application.prooperties` without searching for Polaris doc how to config things. We can even put certain common used configures in it, and comment them out by default, and users can uncomment them when needed.